src/test/java/org/openelisglobal/systemUser/UnifiedSystemUserServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package org.openelisglobal.systemUser;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.login.service.LoginUserService;
+import org.openelisglobal.login.valueholder.LoginUser;
+import org.openelisglobal.systemuser.service.SystemUserService;
+import org.openelisglobal.systemuser.service.UnifiedSystemUserService;
+import org.openelisglobal.systemuser.valueholder.SystemUser;
+import org.openelisglobal.userrole.service.UserRoleService;
+import org.openelisglobal.userrole.valueholder.UserRole;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class UnifiedSystemUserServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private UnifiedSystemUserService unifiedSystemUserService;
+    @Autowired
+    private SystemUserService systemUserService;
+    @Autowired
+    private UserRoleService userRoleService;
+    @Autowired
+    private LoginUserService loginUserService;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSetWithStateManagement("testdata/system-user.xml");
+    }
+
+    @Test
+    public void deleteData() {
+        // showing exiting system users
+        List<SystemUser> systemUsers = systemUserService.getAll();
+        assertTrue(systemUsers.size() > 0);
```

Comment:
Can you add some system users via datasets so that we have some data before delete??